cirq-google/cirq_google/serialization/circuit_serializer_test.py

```diff
@@ -389,8 +389,10 @@ def test_serialize_deserialize_circuit():
     assert serializer.deserialize(proto) == circuit
 
 
-def test_serialize_deserialize_circuit_with_moments_table():
-    serializer = cg.CircuitSerializer(USE_CONSTANTS_TABLE_FOR_MOMENTS=True)
+def test_serialize_deserialize_circuit_with_constants_table():
```

Comment:
Nit: separately test specific behaviors when possible rather than a monolithic block demonstrating multiple behaviors. e.g., prefer tests with titles that can be phased with clear "given", "when", and "then" clauses.

https://testing.googleblog.com/2014/04/testing-on-toilet-test-behaviors-not.html

Reply:
I am not sure we want to do that in this case.  Testing all combinations of feature flags is not really a viable use case, so I am not sure we want to separate them out.  Both of these flags are here so that we don't serialize something that cannot be deserialized on the server-side.  Once both of these changes are deployed, we will switch the defaults and get rid of the flags.  Separating into two tests is just going to mean we will need to clean up more tests later.

(Arguably, we could just have one feature flag I guess, but I think it makes sense to keep them separate even if we will later likely remove both at the same time).